src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -172,19 +188,22 @@ type private DiffixCount(aidsCount) =
           int64 aggContext.AnonymizationParams.Suppression.LowThreshold
 
       match Anonymizer.count aggContext.AnonymizationParams anonContext state with
-      | None -> Integer minCount
-      | Some { AnonymizedSum = value } -> value |> Math.roundAwayFromZero |> int64 |> max minCount |> Integer
+      | None ->
+        Option.iter (getState >> mergeOutliers (contributionsToArray state)) outliersAggregator
```

Comment:
late thought - shouldn't we subtract `minCount` here? I'm guessing this means we're treating all contributions as outliers, but since we're reporting the `minCount`, the flattening should not include this figure.

Reply:
Good point! Reporting a positive value for the count on insufficient entities will result in an upwards distortion of the total after redistribution. But the outliers are also flattened separately before redistribution, which will distort the total downwards. The two effects should cancel each other somewhat. And with the code being too complicated already, I'm reluctant to do anything about it.